### DISCUSSION TITLE
Status color recommendation

### DISCUSSION THREAD

**Discussion by mindytom** [twilio-labs/paste#1044] [Q&A] [answered]
For Event Streams, we have different sink statuses and they are:

Active = sink is functioning and receiving events from Twilio
Initializing = sink has just been created and there are things orchestrating in the background (will only last a few secs)
Validating = a user just inputted a test id and the sink is validating the id (will only last a few secs)  
Error = something is wrong with the sink  

Does Paste have recommendations of what colors to use for these statuses? 
Also, when will the table component be finalized? 

 [https://twilio.invisionapp.com/d/#/console/20911061/440266928/preview](url)

**Comment by TheSisb** [comment, marked as answer]
Hi @mindytom,

> Does Paste have recommendations of what colors to use for these statuses?

Our tokens are contextually named, so you can leverage the correct background color for your use-case. Check out https://paste.twilio.design/tokens. Some examples: `$color-background-success` and `$color-background-warning`

Keep in mind that for accessibility reasons we recommend pairing up colors with symbols or preferably text since color-blind customers may not always be able to tell statuses apart.

> Also, when will the table component be finalized?

Timing on the Table is currently TBD. However, `Alpha` components are ready for you to use in production. Components such as Table are likely to not see any API changes. This is only an alpha due to the lack of documentation and a matching Sketch symbol.  

If you're blocked by something on the Table component, please start a discussion thread.  We're still gathering data on whether the Table component has seen enough use to graduate to beta / prod status.
